### ISSUE TITLE
"Technical Salon" way of meeting each other

### ISSUE THREAD

**Issue by Frijol** [dweb-camp-2019/projects#21]
## Idea
People come into a casual/lounge space and are given name tags. Instruction is that you don't put your name or where you work on that name tag– instead, you list three things that you're currently working on/excited about.

Probably some very mild structuring like "go talk to somebody new about something on their nametag" "ok now go talk to somebody new and tell them about something you learned in your previous conversation"

## Labels
See https://github.com/dweb-camp-2019/projects/labels

Tentatively "Forum" and "Collaboration Tool". More label description would be helpful!

## Skillsets
What skillsets would be useful to develop this idea?

Facilitation, speaking to groups

## Contact
Your chat handle on #dweb-camp-2019-projects:matrix.org

not sure but I'm @kelsey in the bridged Slack

**Comment by benhylau** [comment]
Excellent idea! Would you be able to facilitate **Technical Salon** at Camp? The _Tea Lounge_ may be a good space for this :) do you think more help is needed to host this session?

>More label description would be helpful!

Thanks for this, I re-read it and realized it ain't clear. I have added descriptions. Also feel free to suggest new tags, I was hoping to let people make / assign their own tags but GitHub doesn't let me do that.

**Comment by Frijol** [comment]
Yeah for sure I can facilitate! No help really needed, though it's always nice to have a second person on hand if somebody wants to.

**Comment by benhylau** [comment]
@Frijol thanks for calling in to the [Open Dialogue call](https://github.com/dweb-camp-2019/organizing/blob/master/open-dialogue/notes/open-dialogue-20190529.md) and glad we were able to briefly discuss this :D

I know we'd get one more person to help, but my feeling is someone at Camp would just volunteer to do as it doesn't require a lot of preparation from the co-facilitator? Probably what's more important is someone with a bit of experience facilitating spaces, and I feel we'll find lots of that talent at Camp.

I think this one is pretty ready to be submitted and posted to the website. Can you fill [this form](https://forms.gle/31xCFaDGwEdNdYP16) and I'll publish on website to make it official? Thanks!

**Comment by whanamura** [comment]
You need to bring nametags!

Allen Gunn does a version of this called "Skill Share."  He asks for about 1/3 of the people to announce a skill they can share (help you tell a good story; etc.)  When they are done, the other 2/3rds find a skill they would like to learn.  It's a 1-on-2 teaching.  Then you switch.  Requires a microphone and can be done with as many as 100 people if the room is big enough.  (This would be good in the Medicine Tent.)
